compute/disk-images/deprecate_disk_image.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package snippets
+
+// [START compute_images_set_deprecation_status]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	compute "cloud.google.com/go/compute/apiv1"
+	computepb "cloud.google.com/go/compute/apiv1/computepb"
+	"google.golang.org/protobuf/proto"
+)
+
+// Geg a disk image from the given project
+func deprecateDiskImage(
+	w io.Writer,
+	projectID, imageName string,
+	// deprecationStatus *computepb.DeprecationStatus,
```

Comment:
please move these sample values to below the function, and/or combine with the values below.

Reply:
Whoops, sorry, forgot to clean up a bit 😅